src/components/Icon/Icon.jsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { BsPencilFill, BsGoogle } from "react-icons/bs";
+import { ImBin } from "react-icons/im";
+import { GiHamburgerMenu } from "react-icons/gi";
+import { CgClose } from "react-icons/cg";
+import { BiCalendar } from "react-icons/bi";
+import { MdPlaylistAddCheck, MdOutlineFastfood } from "react-icons/md";
+import classNames from "classnames";
+import styles from "./Icon.module.scss";
+
+const iconNames = {
+  pencil: BsPencilFill,
+  bin: ImBin,
+  hamburger: GiHamburgerMenu,
+  close: CgClose,
+  google: BsGoogle,
+  calendar: BiCalendar,
+  edit: MdPlaylistAddCheck,
+  food: MdOutlineFastfood,
+};
+
+const colorStyles = {
+  blue: "#7bb9fa",
```

Comment:
Alternatively - Maybe we could use some css classes that would give these color to `Icon`? Similarly for font size? Pros - keeping all the code related to styles in css

Reply:
That's good point. Styles should be in scss files if this is possible. I will make appropriate css classes for this :)